nextflow/MaveDB/nf_modules/variant_recoder.nf

```diff
@@ -0,0 +1,35 @@
+process get_hgvsp {
```

Comment:
similar as `split_by_mapping_type()` - we can have separate `utils` modules for them?